patches/minecraft/net/minecraft/entity/LivingEntity.java.patch

```diff
@@ -233,19 +231,18 @@
        }
     }
  
-@@ -1355,6 +1370,11 @@
+@@ -1355,6 +1368,10 @@
     }
  
     public boolean func_225503_b_(float p_225503_1_, float p_225503_2_) {
 +      float[] ret = net.minecraftforge.common.ForgeHooks.onLivingFall(this, p_225503_1_, p_225503_2_);
 +      if (ret == null) return false;
 +      p_225503_1_ = ret[0];
 +      p_225503_2_ = ret[1];
-+
```

Comment:
I'd like to see a test report of all mixins that changed or got offset when updating MC versions...